packages/Select/__tests__/__snapshots__/Select.test.tsx.snap

```diff
@@ -6,6 +6,7 @@ exports[`Select component should be defined and renders correctly (snapshot) 1`]
     class="form-field form-field--select"
   >
     <select
+      aria-label="Please Choose..."
```

Comment:
It would be good to add some more tests for the new scenario e.g. 
- if label set should not have aria-label
- if custom default text set and no label should see default text set

Reply:
done!